test/transactWriteItem.js

```diff
@@ -0,0 +1,1281 @@
+var async = require('async'),
+  helpers = require('./helpers')
+
+var target = 'TransactWriteItems',
+  request = helpers.request,
+  opts = helpers.opts.bind(null, target),
+  assertType = helpers.assertType.bind(null, target),
+  assertValidation = helpers.assertValidation.bind(null, target),
+  assertTransactionCanceled = helpers.assertTransactionCanceled.bind(null, target),
+  assertNotFound = helpers.assertNotFound.bind(null, target)
+
+describe('transactWriteItem', function () {
```

Comment:
This file really needs a review, specifically that correct exceptions are raised in corner cases, in a way that dynalite behaviour matches AWS dynamodb.

I'm flagging it here because GitHub hides this diff by default calling it "too large".